libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -196,6 +196,7 @@ dictionary CheckMessageResponse {
 enum PaymentTypeFilter {
     "Sent",
     "Received",
+    "ReceivedJIT",
```

Comment:
Now that we persist the payer_bolt11 perhaps we don't need the new `ReceivedJIT` type?